### PR TITLE
Initial Docker File and Setup Fix

### PR DESCRIPTION
**Pull request by ashcrow**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+FROM fedora:latest
+MAINTAINER Red Hat, Inc. <container-tools@redhat.com>
+
+ENV MHM_RELEASE v0.1.0-alpha
+ENV PYTHONPATH  /commissaire-http/src/
+
+# Install required dependencies and commissaire-http
```
Comment (mbarnes) [inline Dockerfile]: Could you add a comment here about why `gcc`, `redhat-rpm-config` and `libffi-devel` are temporarily needed?  I'm not sure of the reason myself.  `git` is obviously needed for `git clone`.  Are the others just dependencies?

Comment (ashcrow) [inline Dockerfile]: Correct, they are dependencies needed by pip when installing some packages.

**Comment by ashcrow** [comment]
:arrow_up:

**Comment by ashcrow** [comment]
[[multi-host-mgr] Enhance http server and base](https://trello.com/c/UQ6kgBDm/489-multi-host-mgr-enhance-http-server-and-base)

**Pull request merged**